### PR TITLE
CR-9052 ingress host from cluster

### PR DESCRIPTION
**Pull request by rotem-codefresh**

### CODE REVIEW

File: cmd/commands/common.go [hunk 1]
```diff
@@ -360,44 +356,21 @@ func getKubeContextNameFromUserSelect(cmd *cobra.Command, kubeContextName *strin
...
-		return fmt.Errorf("missing ingress host")
-	}
-
-	ingressHostPrompt := promptui.Prompt{
-		Label: "Ingress host (required)",
-	}
-
-	ingressHostInput, err := ingressHostPrompt.Run()
+func getIngressHostFromCluster(ctx context.Context, opts *RuntimeInstallOptions) error {
+	fmt.Print("Retrieving ingress controller info from your cluster...\n")
```
Comment (ghost) [inline cmd/commands/common.go]: use logger if you want this message to print to the user (in debug or info). no need for fmt.Print

File: cmd/commands/common.go [hunk 2]
```diff
@@ -360,44 +356,21 @@ func getKubeContextNameFromUserSelect(cmd *cobra.Command, kubeContextName *strin
...
-
-	ingressHostInput = strings.TrimSpace(ingressHostInput)
-	if ingressHostInput == "" {
-		return fmt.Errorf("missing ingress host")
+		return fmt.Errorf("failed to get ingress controller info from your cluster: %w", err)
 	}
 
-	die(cmd.Flags().Set("ingress-host", ingressHostInput))
-	*ingressHost = ingressHostInput
+	opts.IngressHost = fmt.Sprintf("https://%s", ingressController.Status.LoadBalancer.Ingress[0].Hostname)
```
Comment (ghost) [inline cmd/commands/common.go]: do we want to maybe check which ports the LoadBalancer is listening on, to decide if we want to use https or http?

**Review by noam-codefresh — APPROVED** [review]

**Pull request merged**